cmd/cloud-api-adaptor/main.go

```diff
@@ -79,8 +79,8 @@ func (cfg *daemonConfig) Setup() (cmd.Starter, error) {
 
 	case "azure":
 		cmd.Parse("azure", os.Args[1:], func(flags *flag.FlagSet) {
-			flags.StringVar(&azurecfg.ClientId, "clientid", "", "Client Id")
-			flags.StringVar(&azurecfg.ClientSecret, "secret", "", "Client Secret")
+			flags.StringVar(&azurecfg.ClientId, "clientid", os.Getenv("AZURE_CLIENT_ID"), "Client Id")
+			flags.StringVar(&azurecfg.ClientSecret, "secret", os.Getenv("AZURE_CLIENT_SECRET"), "Client Secret")
 			flags.StringVar(&azurecfg.TenantId, "tenantid", "", "Tenant Id")
```

Comment:
Hey JT - from https://github.com/confidential-containers/cloud-api-adaptor/pull/284/files it looks like `tenantid` might also be a sensitive field for Azure